deps/tools/common.mk

```diff
@@ -27,7 +27,11 @@ ifneq ($(VERBOSE), 0)
 CMAKE_COMMON += -DCMAKE_VERBOSE_MAKEFILE=ON
 endif
 # The call to which here is to work around https://cmake.org/Bug/view.php?id=14366
+ifeq ($(BUILD_OS),WINNT)
+CMAKE_COMMON += -DCMAKE_C_COMPILER="$(CC_BASE)"
```

Comment:
the file extension is missing. `$$(which $(CC_BASE))` returns `C:/msys64/mingw64/bin/gcc` which doesn't include the file extension. The referenced bug has also been fixed since cmake >= 3.0 which is available on both Cygwin and windows.
Alternatively I can change it to `CMAKE_COMMON += -DCMAKE_C_COMPILER="$$(which $(CC_BASE)).exe"` which works as well.